apps/api/src/app.module.ts

```
@@ -27,8 +27,20 @@ import { ContactModule } from './modules/contact/contact.module';
+        RESEND_FROM: Joi.string()
+          .default('noreply@example.com')
+          .when('RESEND_API_KEY', {
+            is: Joi.string().min(1),
+            then: Joi.string().min(1).required(),
+            otherwise: Joi.string().allow(''),
+          }),
+        RESEND_TO: Joi.string()
+          .default('admin@example.com')
+          .when('RESEND_API_KEY', {
+            is: Joi.string().min(1),
+            then: Joi.string().email().required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)